docs/getting_started.md

```diff
@@ -1,2 +1,32 @@
-# Getting Started
+# Project initiation
 
+You have an idea for an app. You have approval from your ministry and you're excited to start building. However, before getting started, book an onboarding meeting with the [Developer Experience Team](contact.md). This meeting will help us set up your project in the mobile developer accounts. To make the most of our time together, gather the information outlined below.
+
+## Information to prepare for your onboard meeting
+
+| General information | Response |
+|------|-------|
+| Product Owner name<sup>1</sup> | |
```

Comment:
Overall I find this a weird way to format the table and corresponding info, using the superscript.

Reply:
Reformatted the page